### PR TITLE
updated Fefe.xml to reflect new certificates from Let's Encrypt

### LINKED ISSUE #3510: add ruleset for blog.fefe.de
Following rule should add support for https://blog.fefe.de

    <ruleset name="Fefes Blog">
            <target host="blog.fefe.de" />

            <rule from="^http:"
                    to="https:" />
    </ruleset>

### PR DESCRIPTION
**Pull request by sjjh**
to fix #3510 (instead of #3515)

### CODE REVIEW

**Comment by J0WI** [comment]
You can't put two rulesets in one file. Please separate them an link them in comments.

File: src/chrome/content/rules/Fefe.xml [hunk 1]
```diff
@@ -31,17 +30,23 @@
...
+
 <ruleset name="Fefe.de (partial)" platform="cacert" default_off="missing certificate chain">
 
 	<!--	Direct rewrites:
 				-->
-  <target host="blog.fefe.de" />
-	<target host="cvs.fefe.de" />
+  <target host="cvs.fefe.de" />
 	<target host="dl.fefe.de" />
 	<target host="a.mx.fefe.de" />
```
Comment (youdly) [inline src/chrome/content/rules/Fefe.xml]: `cvs` and `a.mx` are still problematic due to missing certificate chain, suggest to remove line 46 & 48.

File: src/chrome/content/rules/Fefe.xml [hunk 2]
```diff
@@ -31,17 +30,23 @@
...
+<ruleset name="Fefe.de (partial)" platform="Lets Encrypt">
+  <target host="blog.fefe.de" />
+        <target host="ptrace.fefe.de" />
+        <target host="www.fefe.de" />
+
+        <rule from="^http:"
+                to="https:" />
+</ruleset>
+
 <ruleset name="Fefe.de (partial)" platform="cacert" default_off="missing certificate chain">
```
Comment (youdly) [inline src/chrome/content/rules/Fefe.xml]: to re-enable this ruleset, remove `platform` & `default_off` argument, so it's just `<ruleset name="Fefe.de (partial)"`.

File: src/chrome/content/rules/Fefe.xml [hunk 3]
```diff
@@ -31,17 +30,23 @@
...
 
 -->
+<ruleset name="Fefe.de (partial)" platform="Lets Encrypt">
+  <target host="blog.fefe.de" />
+        <target host="ptrace.fefe.de" />
+        <target host="www.fefe.de" />
+
+        <rule from="^http:"
+                to="https:" />
+</ruleset>
```
Comment (youdly) [inline src/chrome/content/rules/Fefe.xml]: as previous [comment](https://github.com/EFForg/https-everywhere/pull/3521#issuecomment-165909189) pointed out, remove line 33, 38-40. Then put line 34-36 below line 42. `platform` argument is not necessary in this case.

**Comment by semenko** [comment]
LGTM, thanks!

**Pull request merged**